app/models/package_manager/npm.rb

```diff
@@ -40,7 +40,7 @@ def self.recent_names
     end
 
     def self.project(name)
-      get("http://registry.npmjs.org/#{name.gsub('/', '%2F')}")
+      get("https://registry.npmjs.org/#{name.gsub('/', '%2F')}")
```

Comment:
separate fix: we were getting 301s for NPM projects this whole time! Avoid that and go direct to HTTPS.